src/mapml/layers/TemplatedFeaturesLayer.js

```diff
@@ -45,36 +43,8 @@ export var TemplatedFeaturesLayer =  L.Layer.extend({
           }
         });
       }
-      // this was tricky...recursion alwasy breaks my brain
-      var features = this._features,
-          _pullFeatureFeed = function (url, limit) {
-            return (fetch (url,{redirect: 'follow',headers: headers})
-                    .then( function (response) {return response.text();})
-                    .then( function (text) {
-              mapml = parser.parseFromString(text,"application/xml");
-              var base = (new URL(mapml.querySelector('map-base') ? mapml.querySelector('map-base').getAttribute('href') : url)).href;
-              url = mapml.querySelector('map-link[rel=next]')? mapml.querySelector('map-link[rel=next]').getAttribute('href') : null;
-              url =  url ? (new URL(url, base)).href: null;
-              let nativeZoom = mapml.querySelector("map-meta[name=zoom]") &&
-                +M.metaContentToObject(mapml.querySelector("map-meta[name=zoom]").getAttribute("content")).value || 0;
-              let nativeCS = mapml.querySelector("map-meta[name=cs]") &&
-                      M.metaContentToObject(mapml.querySelector("map-meta[name=cs]").getAttribute("content")).content || "GCRS";
-              features.addData(mapml, nativeCS, nativeZoom);
-              if (url && --limit) {
-                return _pullFeatureFeed(url, limit);
-              }
-            }));
-          };
-        if(map.getZoom() % this._template.step !== 0) {
-            this._onMoveEnd();
-            return;
-        }
-      _pullFeatureFeed(this._getfeaturesUrl(), 10)
-        .then(function() { 
-              map.addLayer(features);
-              map.fire('moveend');  // TODO: replace with moveend handler for layer and not entire map
-        })
-        .catch(function (error) { console.log(error);});
+
+      map.fire('moveend');  // TODO: replace with moveend handler for layer and not entire map
```

Comment:
What would happen if we replaced this event firing with "this.redraw()"?

Reply:
Seemingly nothing changes, same with calling _onMoveEnd directly however not sure why ` map.fire('moveend');` was needed in the first place so needs a bit more digging

Reply:
Needed to fix some bugs: https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/214

Reply:
Seems that the map moveend firing may cover up some issues in other templated layers, so we can put that consideration aside for now.